UKHomeOffice/cerberus-service
# COP-5900 Remove homepage and redirect / to /tasks

## Description
On login (or click to the homepage) users should be taken to `/tasks`

## To Test
- run Cerberus
- signout (if you are signed in already)
- signin
- > you should be automatically taken to `/tasks`
- click on the govuk logo
- > you should be automatically taken to `/tasks`
- click on the Cerberus heading
- > you should be automatically taken to `/tasks`

## Developer Checklist

\* Required
